### DISCUSSION TITLE
pcl2logo的字体是什么

### DISCUSSION THREAD

**Discussion by FangFuJi1991** [Meloong-Git/PCL#1022] [提问] [answered]
如标题所说 :-D

**Comment by miner233** [comment]
冷知识：Logo 不是龙猫做的，龙猫也不知道
但看样子是...微软雅黑？

**Reply by WForst-Breeze** [reply]
不像是，就单说 C 这个字母就和微软雅黑不一样（）

**Comment by WForst-Breeze** [comment, marked as answer]
翻了一圈也没找到100%一样的……qwq 我个人猜测是 **_LatoLatin Medium Regular_** ，是一款免费商用字体
![image](https://user-images.githubusercontent.com/110760354/215005022-15db42d7-2ceb-4ac5-acef-e339da8a5402.png)
(一个简单的预览，此处字宽设置成了98)

**Reply by FangFuJi1991** [reply]
感谢👍
